metadataserver/server.go

```diff
@@ -195,6 +196,9 @@ func (ms *MetadataServer) installIMDSCommonHandlers(ctx context.Context, router
 
 	/* IAM Stuffs */
 	ms.iamProxy.AttachRoutes(metaData.PathPrefix("/iam").Subrouter())
+	if ms.logIAMProxy != nil {
+		ms.logIAMProxy.AttachRoutes(metaData.PathPrefix("/iam/logging").Subrouter())
```

Comment:
Do we still need this code path if we have the header code path?

Reply:
No, removing.